README.md

```diff
@@ -126,9 +126,11 @@ __add_custom_layers__: bool
 __custom_conversion_fuctions__: dict (str: function)  
 	  Specify custom function to be used for conversion for given op.
         User can override existing conversion function and provide their own custom implementation to convert certain ops.
-        example usage: {'Flatten': custom_flatten_converter, 'Exp': exp_converter}
-        `custom_flatten_converter` and `exp_converter` will be invoked instead of internal onnx-coreml conversion implementation for these two Ops.
-        Note that, `custom_conversion_functions` option is different than `add_custom_layers`.
+        Dictionary key must be string specifying unique name of the ONNX Op and value must be a function implementation available in current context.
```

Comment:
is it just the name or also it can be the type?

Reply:
name. Is it still confusing? unique name of the ONNX op?

Reply:
there is a difference, lets say there is an onnx op which has type "convolution" but name is "conv1" or "conv2" etc.
So if the key is "convolution" all conv ops will invoke the function. If the key in dict is "conv1" then only the convolution op with that name will invoke the function.

Reply:
sounds good. Updating comment.
We are currently supporting key as ONNX op name and not type.

I think this behavior is flexible and gives better control. e.g. dev provides custom function for `conv2d` but uses ours `conv1d` and is always welcome to re-use custom function for multiple ops.

Reply:
we need both. Lets say there is an op "one_hot" which coreml spec does not have and the user wants to run their function on all instances of onnx nodes with type "one_hot" then they should not need to provide names for every single one_hot layer (there might be more than 1)

Reply:
it should be a one line change, to call just check whether the op name or type, one of them is matching or not

Reply:
done